buildenv/docker/mkdocker.sh

```diff
@@ -532,32 +543,33 @@ prepare_user() {
 create_user() {
   echo ""
   echo "# Add user home and copy authorized_keys and known_hosts."
-  echo "RUN useradd -ms /bin/bash $user \\"
+  echo "RUN useradd -ms /bin/bash --uid $userid $user \\"
   echo " && mkdir /home/$user/.ssh \\"
   echo " && chmod 700 /home/$user/.ssh"
   echo "COPY authorized_keys known_hosts /home/$user/.ssh/"
-  echo "RUN chown -R $user:$user /home/$user \\"
-  echo " && chmod 600 /home/$user/.ssh/authorized_keys /home/$user/.ssh/known_hosts"
+  echo "RUN chmod 600 /home/$user/.ssh/authorized_keys /home/$user/.ssh/known_hosts \\"
+  echo " && echo \"PS1=\\\"\\u:DOCKER-CONTAINER $ \\\"\" > /home/$user/.bashrc \\"
+  echo " && echo \"ulimit -u 32000\" >> /home/$user/.bashrc \\"
+  echo " && echo \"ulimit -n 2048\" >> /home/$user/.bashrc"
```

Comment:
Why do we care about the shell prompt string?
Much escaping can be avoided by using single quotes (`'`); the `$` should be escaped, though.
The two limits can be changed in one command: `ulimit -n 2048 -u 32000`.

Reply:
I'll play with modifying.  The reason for the shell prompt was so there was an obvious eyecatcher to indicate that the docker container launch worked and that the job is compiling in the container and not the host.  Purely a debugging/confirmation bit of output for the user.

Reply:
In my latest push I cleaned up the ulimit required change for an added user, running as root ulimit value are unlimited so no compile issues are seen.  Also removed PS1 as it not added huge value.